types.ts

```diff
@@ -17,6 +17,12 @@ export interface GlobOptions<Eager extends boolean, AsType extends string> {
    * Custom queries
    */
   query?: string | Record<string, string | number | boolean>
+  /**
+   * Also search for files inside `/node_modules/`. BECAREFUL: this is slow.
+   *
+   * @default false
+   */
+  includeNodeModules?: boolean
```

Comment:
I feel this is a bit too long. How about `all`? Which we say we have a sensible default to ignore some patterns (dot: false and node_modules, etc.) and have `all: true` to grab all possible files?

Reply:
I'd argue that `all` is not expressive enough: it can mean anything. (Personally, I like when I can guess the meaning of code without having to look up documentation.)

Maybe `noIgnore`?

But if you prefer `all`, I'll rename it `all` :-).

> sensible default to ignore some patterns (dot: false and node_modules, etc.)

👍 Will update the PR.

Reply:
PR updated.
 - Renamed `includeNodeModules` to `noIgnore` (but I'm more than happy to rename it to `all` if you prefer).
 - Added `dot: options.noIgnore ? true : false`.

Reply:
Thinking, how about having two options `nodeModules: boolean` and `dot: boolean`? To give the user fine-grained control.